QRHunter/app/src/main/java/com/example/qrhunter/utils/QRCodeVisual.java

```diff
@@ -0,0 +1,129 @@
+package com.example.qrhunter.utils;
+
+public class QRCodeVisual {
+    /**
+     * generates the visual representation of the qrCode
+     */
+    private String hashValue;
+    private String bitValues;
+
+    // for the qrcode visual representation
+    private String visualRepresentation = "";
```

Comment:
Let's make these variables `final` if they're a constant so that they can't be changed